client/state/comments/schema.js

```diff
@@ -0,0 +1,83 @@
+export const commentsSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+-\\d+$': {
+			type: 'array',
+			items: {
+				type: 'object',
+				properties: {
+					ID: { type: 'integer' },
+					URL: { type: 'string' },
+					author: {
+						type: 'object',
+						properties: {
+							ID: { type: 'integer' },
+							URL: { type: 'string' },
+							avatar_URL: { type: 'string' },
+							email: { type: 'boolean' },
+							first_name: { type: 'string' },
+							last_name: { type: 'string' },
+							login: { type: 'string' },
+							name: { type: 'string' },
+							nice_name: { type: 'string' },
+							profile_URL: { type: 'string' },
+							site_ID: { type: 'integer' }
+						}
+					},
+					content: { type: 'string' },
+					date: { type: 'string' },
+					i_like: { type: 'boolean' },
+					like_count: { type: 'integer' },
+					meta: {
```

Comment:
We should probably update that in the data-layer too if we'd like a different shape.